### ISSUE TITLE
news.scripting.com

### ISSUE THREAD

**Issue by scripting** [scripting/Scripting-News#239]
I have had a news product attached to my blog for a long time. 

Today it switched over to new **FeedLand** technology. 

http://news.scripting.com/

If you have questions, here's the place to post them. 

Still diggin!

**Comment by tomcritchlow** [comment]
Would love to get access to play around with it!

**Comment by scripting** [comment]
@tomcritchlow -- I sent a DM on Twitter.

**Comment by Clay-Ferguson** [comment]
We definitely need to revive RSS in a bigger way. It's still struggling to come back after Google and most BigTech platforms abandoned it because it helps make the middle man (BigTech) unnecessary, and putting users in touch directly with the content itself also eliminates the AD stream, to some extent. I'll be looking into "FreeLand" if I can find it.

**Comment by Crul** [comment]
I think I'm looking at this from the wrong angle, because I don't understand what is FeedLand for.

There doesn't seem to be a public RSS Feed I can subscribe to, and, although the individual feed links are available, I cannot find the list of all the feeds. So I don't see an easy way to include content from FeedLand into my RSS reader.

What am I missing?
Thanks

**Comment by Crul** [comment]
Yes, I think my confusion is a mix of the project being in its earliest stage and not fitting my (personal and specific) use of RSS. To me, the main point of RSS is to avoid having to visit multiple (hundreds of) sites to get the latest posts / news. (Unless I'm missing something, in that case, please correct me:) 
If FeedLand were successful and a lot of people published their "curated aggregated news feed", I wuold still have to visit all those sites.

Thanks for the answer.

**Comment by scripting** [comment]
If each of these FeedLand sites also published the OPML subscription list behind it, would that do it for you?

**Comment by Crul** [comment]
Indeed, that would be awesome. I love RSS and it would be great to see its ecosystem grow. Sharing OPMLs is one of the things I miss the most. Thanks for your work!

**Comment by scripting** [comment]
Stay tuned…

**Comment by scripting** [comment]
> personal search engine

good idea!

**Comment by Crul** [comment]
@scripting That was FAST! Thank you very much.

**Comment by dmatchett** [comment]
Exciting stuff -- don't fully understand the code but I love sending feeds hither and yon in an endless search for my own best river. For me currently NNW is a big catchment basin and catalog.

**Comment by scripting** [comment]
@dmatchett -- I hope all this stuff works well with NetNewsWire. A shout-out to Brent and the other developers of that product. 

I see FeedLand as an OPML subscription list generator, and I imagine NNW is a first class consumer of them (and of course a generator).

**Comment by fmcpherson** [comment]
I am wondering what is difference between Feedland and River5?

**Comment by scripting** [comment]
@fmcpherson -- thanks for asking this question. It's a good one, I should know how to answer it. So let's give it a try.

1. It's much more than River5. 

2. It's a browser-based UI for managing and sharing subscriptions. So the users are kind of members of a club. We can all see each others subscription lists. We know what are the most popular feeds. When someone subscribes to a feed it appears in the subscription log. 

3. It's _very easy_ to subscribe to a feed that's already in the database. Radically easy. Hard to imagine how it could be easier. That's very important.

4. I can create a "news product" entirely in the web browser, no need to use an external tool to manage your feeds and categories. Again, as easy as it can be. I iterated over the design of this stuff obsessively. 

5. It stores all the data in a relational database. 

6. So far none of this stuff is in River5. Underneath all this, yes FeedLand does almost all of what River5 does. There are still a few things left to do.

Basically where River5 is a simple feed reading system, this is groupware, collaborative, like Slack it's meant for workgroups. The end result is not entirely dissimilar, but the chance to work with other people is new.

**Comment by scripting** [comment]
This became a blog post, of course. :boom:

http://scripting.com/2022/10/07/161137.html?title=comparingFeedlandToRiver5

**Comment by drewkime** [comment]
Ok to put some bug reports here?

![image](https://user-images.githubusercontent.com/6943880/194951964-9f08e01b-92d4-4443-b0e6-b60e5ddd2a2e.png)

All the icons I see for what look like podcasts show "No enclosure attached to this item". Also, this instance doesn't show the link to the site.

![image](https://user-images.githubusercontent.com/6943880/194952211-9853bf32-8947-4727-8a98-cca7a8ba87cb.png)

This one shows "No date".

**Comment by scripting** [comment]
I don't think those are bugs. 

Here's what that feed looks like to FeedLand.

http://feeder.scripting.com/returnjson?feedurl=http%3A%2F%2Fgizmodo.com%2Findex.xml

As you can see there is no feed-level pubDate. That's what [no date] means.

And that item has no enclosure, that's why it says it has no enclosure.

You're going to have to try harder, if you want to find actual bugs. I'm sure they are there, but these aren't them. ;-)

**Comment by scripting** [comment]
Here's a news product with some podcasts. 

http://dave.podcatch.com/
 
It's possible that the disabled podcast icon is too subtly different from the enabled one.

**Comment by drewkime** [comment]
Ah, so that's from a podcast channel, but without a podcast on this post? What is the difference between enabled and disabled? Sorry if these are super basic questions, I'm a reader not a creator.

**Comment by scripting** [comment]
These are good questions, and are certain to come up again and again. 

If an enclosure icon is enabled that means when you click on it you hear something, because the item has a podcast attached.

It's disabled if there was no podcast attached.

There's no such thing as a "podcast feed" -- just feeds. Some items might not have enclosures. There are no rules about this. 

And a human being, probably me, classified the feed as a podcast -- and sometimes we make mistakes.  I don't know about this case. I don't think you said what the feed was. I can't see the URL in a screen shot. So if it really were a bug report it would have to include enough info so I could see what you're seeing.
